src/py42/exceptions.py

```diff
@@ -54,7 +54,12 @@ class Py42HTTPError(Py42ResponseError):
     """A base custom class to manage all HTTP errors raised by an API endpoint."""
 
     def __init__(self, exception, message=None):
-        message = message or u"Failure in HTTP call {}".format(str(exception))
+        error_message = ""
+        if hasattr(exception.response, "text"):
+            error_message = "\n Error message: {}".format(exception.response.text)
```

Comment:
it would probably be more accurate to call this `Response content` instead of `Error message`, since it might not contain anything, or would often be a JSON object as opposed to a message string.

Reply:
Changed.